skiko/src/commonMain/cpp/headers/common.h

```diff
@@ -0,0 +1,59 @@
+#ifndef SKIKO_COMMON_H
+#define SKIKO_COMMON_H
+
+#include "SkCodec.h"
+#include "SkFontMetrics.h"
+#include "SkFontStyle.h"
+#include "SkImageInfo.h"
+#include "SkMatrix.h"
+#include "SkM44.h"
+#include "SkPaint.h"
+#include "SkRefCnt.h"
+#include "SkRect.h"
+#include "SkRRect.h"
+#include "SkScalar.h"
+#include "SkShaper.h"
+#include "SkString.h"
+#include "SkSurfaceProps.h"
+#include <stdexcept>
+#include "types.h"
+
+jlong packTwoInts(int32_t a, int32_t b);
+
+jlong packIPoint(SkIPoint p);
+
+jlong packISize(SkISize p);
+
+namespace skija {
```

Comment:
Probably we shall rename skija -> skiko eventually.

Reply:
yes, I suppose.